docs/tutorial/part-eight/index.md

```diff
@@ -331,44 +333,44 @@ export const query = graphql`
 `
 ```
 
-The above example is based off the [Gatsby Starter Blog](/starters/gatsbyjs/gatsby-starter-blog/). By passing props to the `<SEO>` component, you can dynamically change the metadata for a post. In this case, the blog post `title` and `excerpt` (if it exists in the blog post markdown file) will be used instead of the default `siteMetadata` properties in your `gatsby-config.js` file.
+Powyższy przykład oparty jest na [Gatsby Starter Blog](/starters/gatsbyjs/gatsby-starter-blog/). Przesyłając props'y do komponentu `<SEO>`, możesz dynamicznie zmieniać metadane dla postu. W tym wypadku zamiast domyślnych właściwości `siteMetadata` z pliku `gatsby-config.js`, zostaną użyte `title` i `excerpt` bloga (jeśli istnieje w pliku markdown).
 
-Now, if you run the Lighthouse audit again as laid out above, you should get close to--if not a perfect-- 100 score!
+Teraz, jeśli ponownie uruchomisz audyt Lighthouse, zgodnie z powyższym opisem, powinieneś zbliżyć się do - jeśli nie idealnego - 100 punktów!
```

Comment:
```suggestion
Teraz, jeśli ponownie uruchomisz audyt Lighthouse, zgodnie z powyższym opisem, powinieneś zbliżyć się do - jeśli nie osiągnąć - 100 punktów!
```
tutaj trochę to jak dla mnie straciło pełny sens po polsku, wydaje mi się, że tak brzmi lepiej